string root_name
int32 num_robots
---
string[] responses
